### DISCUSSION TITLE
Custom ResetPasswordController not working function redirectPath()

### DISCUSSION THREAD

**Discussion by adrianzofcin** [laravel/nova-issues#4786] [Ideas & Feature Requests]
- Laravel Version: 9.28.0
- Nova Version: 4.13.0
- PHP Version: 8.1
- Database Driver & Version: Mysql 5.7.24
- Operating System and Version: Windows 11
- Browser type and version: Google Chrome

### Description:

I tried to override base nova function in ResetPasswordController.

This is base Nova function

```php
public function redirectPath()
    {
        return Nova::path();
    }

```
And I overwrote function to this

```php
public function redirectPath()
    {
        return redirect('/successful-password-change');
    }

```
I registered my controllers to override base nova controllers

```php
/**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
        $this->app->bind(
            \Laravel\Nova\Http\Controllers\LoginController::class,
            LoginController::class,
        );
        $this->app->bind(
            \Laravel\Nova\Http\Controllers\ResetPasswordController::class,
            ResetPasswordController::class,
        );
    }
```

My controllers also extend base nova controllers from vendor

`class ResetPasswordController extends \Laravel\Nova\Http\Controllers\ResetPasswordController`

Problem isn't that it wouldn't register my controllers but function redirectPath() in ResetPasswordController was completely ignored.

I tried this because it supposedly worked in nova 3
Link to the issue: https://github.com/laravel/nova-issues/issues/2098

[issued-files.zip](https://github.com/laravel/nova-issues/files/9525567/issued-files.zip)

**Comment by juse-less** [comment]
You can add custom routes for password resets, authentication, and some more by following [this example](https://github.com/laravel/nova-issues/discussions/3954#discussioncomment-2569596).

Edit: Re-reread the last part of your message, and realised it probably won't solve your issue directly, if the controller is already executed.

**Comment by crynobone** [comment]
Yes, it is currently ignored at the moment.
